src/Spec2-Adapters-Morphic/SpMorphicMillerAdapter.class.st

```diff
@@ -233,10 +233,13 @@ SpMorphicMillerAdapter >> scrollByDeltaPercent: percent [
 { #category : 'private' }
 SpMorphicMillerAdapter >> scrollToShowLastPage [
 
-	self widget defer: [ | size firstMorphIndex |
+	self widget defer: [
+		| size firstMorphIndex |
 		size := innerWidget submorphs size.
-		firstMorphIndex := size - self presenter visiblePages + 1. 
-		self widget showMorph: (innerWidget submorphs at: ((firstMorphIndex max: 1) min: size)) ]
+		firstMorphIndex := size - self presenter visiblePages + 1.
```

Comment:
I would put this line inside the guard... no need on calculate the index if we are not going to use it.